### ISSUE TITLE
Reduce boilerplate for copy views

### ISSUE THREAD

**Issue by Kusand** [Comcast/blueprint#24]
Lots of components are simple copy presentation, showing a title, or a single string, etc. We still have to write a lot of boilerplate for these; you need a ComponentModel extension to wrap your single String, you need a special presenter, etc. It would be nice to have an easier way to do this. Perhaps an annotation on your viewholder that indicates it is a copy view and the id of the textview for the copy; it would then generate a custom model, presenter, and static method for creating the component?

**Comment by dapp** [comment]
I don't think we should be generating component code, no matter how simple they are.  If the component has _any_ presentation logic, even if that logic is simply setting text on a single view, you have to write a presenter for it.  If you want to write a companion class to simplify the construction of a component that obeys certain criteria, for instance that it only has a single text view, that belongs in your app, not in the framework.

However, if you simply want to use a static layout that needs NO presentation logic at all, and no model, I've provided an alternate Component ctor to simplify that in https://github.com/Comcast/blueprint/pull/25
